zipline/test_algorithms.py

```diff
@@ -703,8 +725,8 @@ def handle_data(self, data):
                 self.return_nan.handle_data(data))
         else:
             nan_data = deepcopy(data)
-            for sid in nan_data.iterkeys():
-                nan_data[sid].price = np.nan
+            for sidint in nan_data.iterkeys():
```

Comment:
Makes sense. I didn't touch the logic here, really, just the variable name, but I've put in the change you suggested.